queries.sql

```diff
@@ -12,4 +12,55 @@ SELECT * FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name != 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;   // const user = userData ? JSON.parse(userData) : null
+
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(100);
```

Comment:
👏 Good job on adding the new `species` column with a string type! 👏 However, there's a small adjustment needed:

- [ ] **Organizing Queries in `schema.sql`:** According to project requirements, any modifications to the table structure should be saved in the `schema.sql` file. Please consider moving the `ALTER` query for the new `species` column from `queries.sql` to the appropriate section in the `schema.sql` file.

By ensuring the query is in the correct file, you'll maintain consistency with project guidelines. Keep up the great work! 🚀📄